### PR TITLE
Added filter rtmedia_media_tab_name for RTMEDIA_MEDIA_LABEL constant

### PR DESCRIPTION
**Pull request by ravatparmar**

### CODE REVIEW

File: app/main/controllers/template/rtmedia-actions.php [hunk 1]
```diff
@@ -692,7 +692,7 @@ function rtm_album_media_count() {
 
 	if ( isset( $rtmedia_album_count_status ) && $rtmedia_album_count_status['status'] ) {
 		?>
-		<div class="rtmedia-album-media-count" title="<?php echo rtmedia_album_mediacounter() . RTMEDIA_MEDIA_LABEL; ?>">
+		<div class="rtmedia-album-media-count" title="<?php echo rtmedia_album_mediacounter() . ' ' . RTMEDIA_MEDIA_LABEL; ?>">
```
Comment (SID177) [inline app/main/controllers/template/rtmedia-actions.php]: Why you haven't used escaping for title attribute?

Comment (ravatparmar) [reply app/main/controllers/template/rtmedia-actions.php]: @SID177 For maintaining consistency between photos, videos, and albums.

Comment (SID177) [reply app/main/controllers/template/rtmedia-actions.php]: Consistency for escaping?

Comment (ravatparmar) [reply app/main/controllers/template/rtmedia-actions.php]: Done... I read sentence wrongly.

**Pull request merged**